ftw/mail/utils.py

```diff
@@ -295,7 +295,12 @@ def get_filename(msg, content_type=None):
         if content_type is None:
             content_type = msg.get_content_type()
         if content_type == "message/rfc822":
-            filename = "attachment.eml"
+            unwrapped = unwrap_attached_msg(msg)
+            subject = get_header(unwrapped, "Subject") or "attachment"
```

Comment:
Shouldn't we translate attachment?

Reply:
hum should we? It will basically be in the language of the person uploading the mail. And this should basically never happen.

Reply:
IMO having it in user language would be fine. But if you don't think it is necessary that also works for me 👍.

Reply:
I've updated to use the default subject, translated, as filename